Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -1690,6 +1690,14 @@ void CSSPrimitiveValue::collectComputedStyleDependencies(ComputedStyleDependenci
         dependencies.properties.appendIfNotContains(CSSPropertyFontSize);
         dependencies.properties.appendIfNotContains(CSSPropertyLineHeight);
         break;
+    case CSSUnitType::CSS_CQW:
+    case CSSUnitType::CSS_CQH:
+    case CSSUnitType::CSS_CQI:
+    case CSSUnitType::CSS_CQB:
+    case CSSUnitType::CSS_CQMIN:
+    case CSSUnitType::CSS_CQMAX:
+        dependencies.containerDimensions = true;
+        break;
     case CSSUnitType::CSS_CALC:
         m_value.calc->collectComputedStyleDependencies(dependencies);
         break;
```

Comment:
@anttijk Would it be a good idea in a follow up patch to replace `default:` with all the units enumerated? and `ASSERT_UNREACHED` as default case? 

Seems like one can easily forget to add a unit here. (I'm thinking of new units like rex/rcap/etc. that Chrome has implemented)

Reply:
Good idea, I'll do it in this patch.